pkg/reconciler/revision/resources/deploy.go

```diff
@@ -99,6 +106,23 @@ func makePodSpec(rev *v1.Revision, loggingConfig *logging.Config, tracingConfig
 
 	podSpec := BuildPodSpec(rev, append(BuildUserContainers(rev), *queueContainer))
 
+	if features.RuntimeV1VarLogVolume == config.Enabled {
```

Comment:
What happens when /var/log collection is enabled?
Is the goal to keep the collection part but disable the volume mounting part?
I though we wanted to get rid of both.

Reply:
> What happens when /var/log collection is enabled?

It doesn't work - I should probably point that out in the feature flag comments as a WARNING

> Is the goal to keep the collection part but disable the volume mounting part? I though we wanted to get rid of both.

Last I checked we're waiting on some docs/website changes prior to dropping monitoring - https://knative.slack.com/archives/CAN0F23QW/p1600715808001100

Reply:
Ok, then why not use the var-log-collection flag directly if they are incompatible?
We could just notify people of the deprecation ([it is already `false` by default](https://github.com/knative/serving/blob/master/config/core/configmaps/observability.yaml#L44)), and then drop the feature in the next release?
Isn't this way simpler?

Reply:
I was approaching it this way since it breaks the runtime contract.

Reply:
The contract is going to be broken no matter what, isn't it?
What I am suggesting requires no more additional code.
Simply communication, wait and code deletion.

Reply:
Moved the discussion to slack: https://knative.slack.com/archives/CA4DNJ9A4/p1601410619046000